index.html

```diff
@@ -6,24 +6,38 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>PATRICK MWANGI PORTIFOLIO</title>
     <link rel="stylesheet" href="style.css">
-		<link rel="stylesheet" href="desktop.css">
+	<link rel="stylesheet" href="desktop.css">
+	
 </head>
 <body>
 	
 	<!-- header -->
 	<header>
-		<div class="logo">Patrick Wanjiru</div>
-		<ul>
-			<li><a href="">Portifolio</a></li>
-			<li><a href="">Contact</a></li>
-			<li><a href="">About</a></li>
-
-		</ul>
-	</header>
+        <nav class="navbar">
+            <a href="#" class="head">Patrick</a>
+            <ul class="nav-menu">
+                <li class="nav-item">
+                    <a href="#headline " class="nav-link">Portfolio</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#about" class="nav-link">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#contact" class="nav-link">Contact</a>
+                </li>
+            </ul>
+            <div class="nav-icon">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
```

Comment:
- [OPTIONAL] I suggest putting this closing button on the popup to make it doesn't go when we scroll the page :recycle: 
- [OPTIONAL] I think it would be better to make the body overflow scroll when the popup is on :recycle: